ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -221,7 +221,7 @@ def wrapped_connection(
         """
         connection = connect_method(*args, **kwargs)
         self.get_connection_attributes(connection)
-        return TracedConnectionProxy(connection, self)
+        return get_traced_connection_proxy(connection, self)
```

Comment:
Any reason for these wrapper functions?

Reply:
yes - see my above comment. tl;dr: Can't assign new properties to the psycopg2 connection object